federation/sssd/src/main/java/org/freedesktop/dbus/AbstractConnection.java

```diff
@@ -1032,8 +1032,8 @@ else if (m instanceof MethodReturn)
                     transport.mout.writeMessage(new Error(m, e));
                 } catch (IOException IOe) {
                     if (EXCEPTION_DEBUG && Debug.debug) Debug.print(Debug.ERR, IOe);
-                } catch (DBusException IOe) {
-                    if (EXCEPTION_DEBUG && Debug.debug) Debug.print(Debug.ERR, e);
+                } catch (DBusException DBe) {
```

Comment:
I can't see how this changes anything. "IOe" is only visible within the catch block, so there's two distinct instances of it depending on what catch block it uses. Changing the name shouldn't affect anything.

Reply:
If the next line is:

```
if (EXCEPTION_DEBUG && Debug.debug) Debug.print(Debug.ERR, IOe);
```

the stacktrace of `IOe` will be printed and no problems. However, now it is ignored because of:

```
if (EXCEPTION_DEBUG && Debug.debug) Debug.print(Debug.ERR, e);
```

There is no need to rename the variable `IOe` but I think it is not appropriate to use `IOe` as the variable name of `DBusException`.